File: src/main/java/net/softbell/bsh/iot/dto/bshp/v1/NodeInfoV1DTO.java
package net.softbell.bsh.iot.dto.bshp.v1;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : BSHPv1 전용 Node Info DTO
 */
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class NodeInfoV1DTO
{
	private String uid;
	private byte controlMode;
	private String nodeName;
}

File: src/main/kotlin/net/softbell/bsh/iot/dto/bshp/v1/NodeInfoV1Dto.kt
package net.softbell.bsh.iot.dto.bshp.v1

/**
 * @author : Bell(bell@softbell.net)
 * @description : BSHPv1 전용 Node Info DTO
 */
data class NodeInfoV1Dto (
    var uid: String,
    var controlMode: Byte,
    var nodeName: String,
    var version: String
)